extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/EngineConfigurationsBuildItem.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.qute.deployment;
+
+import java.util.Collection;
+
+import org.jboss.jandex.ClassInfo;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+final class EngineConfigurationsBuildItem extends SimpleBuildItem {
```

Comment:
Can this class be used in other extensions to register their own *Resolvers?

Reply:
Nope. Other extensions could just place a `ValueResolver` impl annotated with `@EngineConfiguration` in the runtime module and make it part of the bean archive index (e.g. via `AdditionalBeanBuildItem`).